### PR TITLE
JavaScript API PS

### PR DESCRIPTION
**Pull request by royjhan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -187,6 +187,14 @@ ollama.embeddings(request)
   - `options` `<Options>`: (Optional) Options to configure the runtime.
 - Returns: `<EmbeddingsResponse>`
 
+### process
+
+```javascript
+ollama.process()
```
Comment (jmorganca) [inline README.md:193]:
```suggestion
ollama.ps()
```

File: README.md [hunk 2]
```diff
@@ -187,6 +187,14 @@ ollama.embeddings(request)
   - `options` `<Options>`: (Optional) Options to configure the runtime.
 - Returns: `<EmbeddingsResponse>`
 
+### process
```
Comment (jmorganca) [inline README.md:190]:
```suggestion
### ps
```

Ideally we describe this a bit but to keep it consistent

**Comment by HopperElec** [comment]
I agree that the name should be named consistently with other functions, but I personally don't think functions should need to be named the same as the API endpoint because surely the purpose of this library is to abstract away from the API. Obviously, though, this would be a breaking change, so it would either need to be a major change or act as aliases (perhaps with old ones being deprecated).

File: examples/list/list.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import ollama from '../../src/browser'
```
Comment (BruceMacD) [inline examples/list/list.ts:1]: let's omit this example for now

**Review by BruceMacD — COMMENTED** [review]
Looking good, thanks for adding this. I missed that this would need to be updated.

**Pull request merged**